pkg/operator/ceph/config/config.go

```diff
@@ -58,6 +62,9 @@ const (
 
 	// CephGroup is the Linux Ceph groupname
 	CephGroup = "ceph"
+
+	// CephConfFileName is the default name of the Ceph config file.
+	CephConfFileName = "ceph.conf"
```

Comment:
Why not using https://github.com/rook/rook/pull/3573/files#diff-1436fdad6d9a57ca53d46fa7b9879a2bR54?